content/en/docs/demo/telemetry-features/manual-span-attributes.md

```diff
@@ -5,7 +5,7 @@ aliases: [manual_span_attributes, ../manual-span-attributes]
 
 This page lists the manual Span Attributes used throughout the demo:
 
-## AdService
+## Ad Service
```

Comment:
Optional: "Service" could be removed from the headings on this page for consistency because some components, like the Load generator, are without it. This way, we wouldn't need to differentiate components that are considered services from those that aren't.